wcpredictor/src/bpl_interface.py

```diff
@@ -190,20 +212,62 @@ def get_fixture_goal_probabilities(
         goals = np.arange(0, max_goals + 1)
         probs = {}
         simulated_scores = []
-        # if fixture_teams is not passed, we just predict for all games in self.fixtures by default
+        # if fixture_teams not passed, predict for all games in self.fixtures by default
         if fixture_teams is None:
             fixture_teams = self.get_fixture_teams()
         Team_1, Team_2 = zip(*fixture_teams)
         Team_1_conference = [self.confed_dict[team] for team in Team_1]
         Team_2_conference = [self.confed_dict[team] for team in Team_2]
         venue = np.ones(len(Team_1))
         for i in range(len(fixture_teams)):
-            home_team_goal_prob = self.model.predict_score_n_proba(
-                goals, Team_1[i], Team_2[i], Team_1_conference[i], Team_2_conference[i], home=False, neutral_venue=venue[i]
-            )
-            away_team_goal_prob = self.model.predict_score_n_proba(
-                goals, Team_2[i], Team_1[i], Team_2_conference[i], Team_1_conference[i], home=False, neutral_venue=venue[i]
-            )
+            if isinstance(self.model, NeutralDixonColesMatchPredictorWC):
+                home_team_goal_prob = self.model.predict_score_n_proba(
+                    goals,
+                    Team_1[i],
+                    Team_2[i],
+                    Team_1_conference[i],
+                    Team_2_conference[i],
+                    home=False,
```

Comment:
should these always be false? shouldn't we have these True when it isn't a neutral venue?

Reply:
something like `home=!venue[i]`